Source/Scene/Scene.js

```diff
@@ -1471,8 +1492,12 @@ define([
             executeCommand(skyAtmosphereCommand, scene, context, passState);
         }
 
+        if (defined(sunComputeCommand)) {
```

Comment:
The sun code always generated its texture during its first update, I just kept it the same.